### ISSUE TITLE
ParenthesesAsGroupedExpression should ignore multiple argument calls

### ISSUE THREAD

**Issue by deivid-rodriguez** [bbatsov/rubocop#2145]
The cop registers an offense for something like this:

```
assert_equal (0..1.9), acceleration.domain
```

It surprised me since the call is not ambiguous because it has multiple parameters.

**Comment by jonas054** [comment]
Ruby 1.9.3 gives the same warning if you run `ruby -wc`, but in Ruby 2.1 and 2.2 it's been removed. I think you're right. We could remove this warning in RuboCop too.

**Comment by deivid-rodriguez** [comment]
@jonas054 You mean remove the cop altogether or just the case I'm mentioning? What's the preferred style, `MyClass.new(0..1)` or `MyClass.new((0..1))`?

Thanks!

**Comment by jonas054** [comment]
Just change the cop so it doesn't report when the grouped expression is the first argument in a multiple argument call, is what I meant.

`MyClass.new(0..1)` is better IMO. We're used to see parentheses around ranges, but that's just because they're often needed due to operator precedence. Not so in this case.

**Comment by deivid-rodriguez** [comment]
Nice. That does look like Ruby. So should we leave the cop as it is for now?

**Comment by jonas054** [comment]
No, leave this issue open and someone will, if @bbatsov agrees, close it with a pull request that modifies the behavior of `ParenthesesAsGroupedExpression`.

**Comment by deivid-rodriguez** [comment]
:+1: 

Thanks @jonas054
